trade-finance-manager-api/graphql-query-tests/mock-deal.js

```diff
@@ -238,6 +238,7 @@ const MOCK_DEAL = {
     submissionDetails: {
       supplierPartyUrn: 'MOCK supplierPartyUrn',
     },
+    activities: [],
```

Comment:
I don't think we need this - looks like the `submissionDetails` is actually invalid as not in the graphQL schema